infrastructure/src/main/resources/logback.prod.xml

```diff
@@ -50,7 +50,7 @@
     <root level="info">
         <appender-ref ref="async_file"/>
     </root>
-    <logger additivity="false" level="info" name="org.corfudb.metricsdata">
```

Comment:
@PavelZaytsev this is a bug right? PR #2841 will disable the server metrics.

Reply:
This line should be 'debug' because in MeterRegistryInitializer we pass logger consumer as logger.debug. It needs to be whatever the consumer we are passing to create a registry